datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal.tsx

```diff
@@ -57,13 +63,53 @@ export const EditOwnersModal = ({
     onCloseModal,
     refetch,
     entityType,
+    onOkOverride,
+    title,
+    defaultValues,
 }: Props) => {
     const entityRegistry = useEntityRegistry();
+
+    // Renders a search result in the select dropdown.
+    const renderSearchResult = (entity: Entity) => {
+        const avatarUrl =
+            entity.type === EntityType.CorpUser
```

Comment:
might be easier to say `entity.type === EntityType.CorpUser && (entity as Corpuser).editableProperties?.pictureLink`